pages/22.What's New/docs.en.md

```diff
@@ -13,6 +13,32 @@ taxonomy:
 ----------------------------
 In this section we'll highlight the new features for each major release of Mautic.  They are in version order beginning with the latest.
 
+##Version 3.0
+Mautic 3.0 introduces significant features and a number of enhancements. They are outlined as follows:
```

Comment:
This release does not include any features or enhancements, so this might be a bit misleading.

```suggestion
Mautic 3.0 introduces significant changes to the underlying application along with some changes to the minimum version requirements to run Mautic. They are outlined as follows:
```